docs/mentoring.md

```diff
@@ -13,14 +13,25 @@ Mentors and mentees must abide by the [code of conduct](https://github.com/cncf/
 
 ## How To Get A Mentor
 
-Create [a mentoring request](https://github.com/argoproj/argo-workflows/issues/new/choose) with the title "I would like a mentor" and answer the questions in the issue template.
-Note that this is only for general mentoring. For Google Summer of Code, please **do not** create a mentoring request and find the instructions in the relevant section below.
+We establish and end mentor/mentee relationships using issues. The mentee creates the issue and the mentor is assigned the issue. The relationship ends when the issue is closed.
 
-We'll find the right mentor for you.
+To set expectations:
+
+1. Create [a mentoring request](https://github.com/argoproj/argo-workflows/issues/new/choose) with the title "I would like a mentor" and answer the questions in the template.
```

Comment:
We can also use this link? https://github.com/argoproj/argo-workflows/issues/new?assignees=&labels=mentoring%2Ctriage&template=mentoring_request.md